test/e2e/framework/rs_util.go

```diff
@@ -119,3 +121,55 @@ func RunReplicaSet(config testutils.ReplicaSetConfig) error {
 	config.ContainerDumpFunc = LogFailedContainers
 	return testutils.RunReplicaSet(config)
 }
+
+func NewReplicaSet(name, namespace string, replicas int32, podLabels map[string]string, imageName, image string) *extensions.ReplicaSet {
+	return &extensions.ReplicaSet{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "ReplicaSet",
+			APIVersion: "extensions/v1beta1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Namespace: namespace,
+			Name:      name,
+		},
+		Spec: extensions.ReplicaSetSpec{
+			Selector: &metav1.LabelSelector{
+				MatchLabels: podLabels,
+			},
+			Replicas: &replicas,
+			Template: v1.PodTemplateSpec{
+				ObjectMeta: metav1.ObjectMeta{
+					Labels: podLabels,
+				},
+				Spec: v1.PodSpec{
+					Containers: []v1.Container{
+						{
+							Name:  imageName,
+							Image: image,
+						},
+					},
+				},
+			},
+		},
+	}
+}
+
+func LogReplicaSet(rs *extensions.ReplicaSet) {
```

Comment:
@enisoc Oh, they are not needed for current codes, but can be helpful in future. I decided to leave them intact because they can be helpful in future. Same with other functions such as `DeleteReplicaSet` above.

I believe you meant to remove them. I will remove them. Tell me if I shouldn't remove them.